src/index.js

```diff
@@ -574,7 +677,12 @@ export default function (babel) {
     }
   }
 
+  // XXX: source mapping for inserted imports? not sure how to handle.
+  // guessing path here is just the top of the body which is maybe fine?
```

Comment:
I messed with this a little bit. `nodeBefore` is probably the right answer technically since any error here is pointing to generated code at the top of the file. However, it does make Babel's pretty printer collapse all the imports into a non-whitespaced clump.

Not sure if we care.